rollingpin/deploy.py

```diff
@@ -169,6 +177,10 @@ def signal_handler(sig, _):
                             except KeyError:
                                 raise ComponentNotBuiltError(component)
                             deploy_command.append(deploy_ref)
+
+                    # wait on ready command
+                    if not self.dangerously_fast:
```

Comment:
what will it do if the service wasn't restarted?

in general, we need to be careful of emergency situations when adding stuff to the deploy tools.